Clarify that stdout exporter format is unspecified (#4418)
open-telemetry/opentelemetry-specification: specification/logs/sdk_exporters/stdout.md

**Status**: [Stable](../../document-status.md)

"Standard output" LogRecord Exporter is a [LogRecord
Exporter](../sdk.md#logrecordexporter) which outputs the logs to stdout/console.

> The exporter’s output format is unspecified and may vary between

Reviewer: ```suggestion
The exporter's output format is unspecified and may vary between
```

> The exporter’s output format is unspecified and may vary between

Reviewer 2: ```suggestion
The exporter’s output format is unspecified and can vary between
```

This ensures future readers are not confused if this was mistakenly not made a keyword.

> implementations. Documentation SHOULD caution users about this. The following

Reviewer: ```suggestion
implementations. Documentation SHOULD warn users about this. The following
```

wording is recommended:

“This exporter is intended for debugging and learning purposes. It is not

> recommended for production use. The output format is not standardized and may

Reviewer 2: ```suggestion
recommended for production use. The output format is not standardized and can
```

> “This exporter is intended for debugging and learning purposes. It is not
> recommended for production use. The output format is not standardized and may
> change at any time.”

Reviewer: ```suggestion
"This exporter is intended for debugging and learning purposes. It is not
recommended for production use. The output format is not standardized and may
change at any time."
```

"If a standardized format for exporting logs to stdout is desired, consider using
the [File Exporter](../../protocol/file-exporter.md). However, please note that

> the File Exporter is experimental and may not be available in all languages."

Reviewer: It'll be strange is a specific language implementation mentioned "all languages" in its doc.

> “This exporter is intended for debugging and learning purposes. It is not
> recommended for production use. The output format is not standardized and may
> change at any time.”
> 
> "If a standardized format for exporting logs to stdout is desired, consider using
> the [File Exporter](../../protocol/file-exporter.md). However, please note that
> the File Exporter is experimental and may not be available in all languages."

Reviewer 2: ```suggestion
> This exporter is intended for debugging and learning purposes. It is not
> recommended for production use. The output format is not standardized and may
> change at any time.
>
> If a standardized format for exporting logs to stdout is desired, consider using
> the [File Exporter](../../protocol/file-exporter.md). However, please note that
> the File Exporter is experimental and may not be available in all languages.
```
